tests/functional/test_fit_per_framework.py

```diff
@@ -716,8 +716,8 @@ def test_fit_sh(
                 env["CLASS_LABELS_FILE"] = f.name
 
         if parameters:
-            # I don't understand why hypersearch params are taken from a separate json
-            # instead of the model-metadata in python3_sklearn_binary_hyperparameters
+            # model-metadata configs which values are allowed for hyperparameters.
```

Comment:
can we just remove this comment?  It doesn't add new information.

Reply:
removed